index.html

```diff
@@ -699,6 +699,70 @@ <h2>
     const observer = new ComputePressureObserver(pressureChange);
     observer.observe("cpu");
   </pre>
+  <p>
+    In the following example, we want to demonstrate the usage of {{ComputePressureObserver/takeRecords()}},
+    by retrieving the remaining |entries| accumulated since the last callback.
+  </p>
+  <p>
+    It is recommended to do so before {{ComputePressureObserver/disconnect()}}, otherwise {{ComputePressureObserver/disconnect()}} will clear them and they will be lost.
+  </p>
+  <pre class="example js" title="Usage of takeRecords">
+    function pressureChange(entries) {
+      // do something with entries
```

Comment:
Shall we concat entries instead?

```js
let entries = [];
function pressureChange(changes) {
    entries = entries.concat(changes);
    // Do something with entries...
}

...

entries = entries.concat(observer.takeRecords());
```

Reply:
It s all depending on what you re going to do with the entries.
Maybe the last one is only relevant to you.